js/controller.js

```diff
@@ -642,6 +642,8 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
         this.state.upNextInfo.delayedSetEmbedCodeEvent = true;
       }
       else {
+        this.state.screenToShow = CONSTANTS.SCREEN.LOADING_SCREEN;
+        this.renderSkin();
```

Comment:
Nothing would happen, I just did not want to separate two related lines - setting the screen and rendering. I can move it below if you think it will be better.